core/shared/src/main/scala/sigma/util/CollectionUtil.scala

```diff
@@ -41,6 +41,18 @@ object CollectionUtil {
     result
   }
 
+  /** Concatenate all the arrays in the sequence. */
+  def concatArrays[T: ClassTag](seq: Traversable[Array[T]]): Array[T] = {
```

Comment:
Adding here method for one test just is overkill, for tests ++ is okay

Reply:
removed

Reply:
push the code